src/mpc-hc/mpcresources/PO/mpc-hc.ar.menus.po

```diff
@@ -190,6 +190,14 @@ msgctxt "ID_VIEW_ZOOM_AUTOFIT_LARGER"
 msgid "Auto Fit (&Larger Only)"
 msgstr "احتواء تلقائي (حجم أكبر فقط)"
 
+msgctxt "ID_VIEW_ZOOM_ADD"
```

Comment:
You should not modify the .po files. Translations will be taken care of by our translators at Transifex website.